packages/perseus-editor/src/components/coordinate-pair-input.tsx

```diff
@@ -47,38 +74,56 @@ const CoordinatePairInput = (props: Props) => {
 
     return (
         <View>
+            {/* Errors */}
+            {xRangeError && (
+                <LabelMedium style={styles.errorText}>
+                    {xRangeError}
+                </LabelMedium>
+            )}
+            {yRangeError && (
+                <LabelMedium style={styles.errorText}>
+                    {yRangeError}
+                </LabelMedium>
+            )}
+
+            {/* Coordinate input fields */}
             <View style={[styles.row, styles.spaceUnder]}>
                 <LabelMedium tag="label" style={styles.row}>
-                    {labels ? labels[0] : "x coord"}
+                    {labels[0]}
 
                     <Strut size={spacing.xxSmall_6} />
                     <TextField
                         type="number"
                         value={coordState[0]}
                         min={range ? range[0][0] : undefined}
                         max={range ? range[0][1] : undefined}
-                        onChange={(newValue) => handleCoordChange(newValue, 0)}
+                        onChange={(newValue) =>
+                            handleCoordChange(newValue, 0)
+                        }
```

Comment:
I tried using WB TextField's `validate` and `onValidate` props here, but I couldn't get it to update when I wanted it to.